integration_tests/redis/test.sh

```diff
@@ -15,7 +16,11 @@ for cfg in $configs; do
     CONTAINER_NAME=zgrab_redis_${cfg}
     echo "redis/test: Testing $CONTAINER_NAME"
     CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis > "$ZGRAB_OUTPUT/redis/${cfg}-normal.json"
+    CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis --mappings "$CONTAINER_DIR/mappings.json" > "$ZGRAB_OUTPUT/redis/${cfg}-normal-mappings.json"
+    CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis --custom-commands "$CONTAINER_DIR/extra-commands.json" > "$ZGRAB_OUTPUT/redis/${cfg}-normal-extra.json"
     CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis --inline > "$ZGRAB_OUTPUT/redis/${cfg}-inline.json"
+    CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis --inline --mappings "$CONTAINER_DIR/mappings.yaml" > "$ZGRAB_OUTPUT/redis/${cfg}-inline-mappings.json"
+    CONTAINER_NAME=$CONTAINER_NAME $ZGRAB_ROOT/docker-runner/docker-run.sh redis --inline --custom-commands "$CONTAINER_DIR/extra-commands.yaml" > "$ZGRAB_OUTPUT/redis/${cfg}-inline-extra.json"
```

Comment:
Minor comment:  Should probably "" all `$ZGRAB_ROOT/docker-runner/docker-run.sh` calls, in case someone is a masochist and has a directory name with a space in their development path.

Reply:
Updated, though if someone has spaces in their dev path, they deserve the pain that would've come with debugging this

Reply:
The template could probably use an overhaul; fixing quoting issues, using better detection of docker images / containers (not relying on grep), and adding `set -u` all seem like obvious improvements.